docs/swaggerDefinition.js

```diff
@@ -579,6 +579,34 @@ const swaggerOptions = {
             }
           }
         },
+        exchange: {
+          type: 'object',
+          properties: {
+            bankName: {
+              type: 'string'
+            },
+            bankId: {
+              type: 'string'
+            },
+            isActive: {
+              type: 'boolean'
+            },
+            quantity: {
+              type: 'number'
+            },
+            currency: {
+              type: 'object',
+              properties: {
+                denaru: {
```

Comment:
`denaru`? 🤔 
Can we have it from constants?

Reply:
Forgot :sweat: